code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -221,6 +221,19 @@
 		M.Sleeping(30)
 	M.drowsyness = max(M.drowsyness, 30)
 
+/datum/reagent/mothpheromones
+	name = "Moth pheromones"
+	id = "mothpheromones"
+	description = "Basicly the liquid concept of fluttering towards a lamp."
+	reagent_state = LIQUID
+	color = "#8fddf7" // rgb: 143, 221, 247
```

Comment:
input a `taste_description` of your liking, or it will end up tasting like the metaphorical salts.